src/main/java/com/project/springbootproject/dto/order/OrderDtoStatus.java

```diff
@@ -0,0 +1,9 @@
+package com.project.springbootproject.dto.order;
+
+import com.project.springbootproject.model.Order;
+import lombok.Data;
+
+@Data
+public class OrderDtoStatus {
+    private Order.Status status;
+}
```

Comment:
Why you create many small dto classes?

Reply:
At the moment I find it more convenient to work with the returned type in this way